src/test/java/copy_credentials/CopyCredentialsInteractorTest.java

```diff
@@ -0,0 +1,52 @@
+package copy_credentials;
+
+import java.awt.Toolkit;
+import java.awt.datatransfer.DataFlavor;
+import java.awt.datatransfer.UnsupportedFlavorException;
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+
+import service.copy_credentials.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
```

Comment:
Remove * imports as it pollutes global namespace. You can see some of the crypto/*Test.java files for the exact imports that don't require *.